src/js/chat.js

```diff
@@ -505,7 +501,7 @@ sock.onmessage = function (e) {
             case "332":
                 // Topic, display?
                 break;
-            // Part and quit both need to be handled the same way in our case - a user left the room
+                // Part and quit both need to be handled the same way in our case - a user left the room
```

Comment:
It's visual studio's formatting. It's a bad habit of mine to do ctrl a before doing it, I need to remember to select only the part I am working on. I agree that the way it was before, I didn't change them because I wanted to.